src/settings.rs

```diff
@@ -141,6 +146,7 @@ impl Settings {
                     file_diff: Some(PROMPT_TO_SUMMARIZE_DIFF.to_string()),
                     commit_summary: Some(PROMPT_TO_SUMMARIZE_DIFF_SUMMARIES.to_string()),
                     commit_title: Some(PROMPT_TO_SUMMARIZE_DIFF_TITLE.to_string()),
+                    lang: Some("ENGLISH".to_string()),
```

Comment:
Use the standard two-letter codes here

See: https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes